scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -12,6 +12,11 @@ export KUBEADMIN_PASSWORD=`cat $KUBEADMIN_PASSWORD_FILE`
 set -x
 export CI="prow"
 export PRNO="$(jq .refs.pulls[0].number <<< $(echo $JOB_SPEC))"
+
+mkdir -p $HOME/.ssh && chmod 0700 $HOME/.ssh
+cp $KAM_SSH_PRIVATE_KEY_FILE $HOME/.ssh/
+chmod +x $HOME/.ssh/id_rsa
+
```

Comment:
I wonder if the test can point to a .ssh folder somewhere.   I am not worry when I see $HOME/.ssh is touched.

Reply:
```
INFO[2021-04-26T10:56:48Z] Command stderr:  ✗  failed to create the gitops repository: "https://github.com/*******/taxi-229.git": failed to push bootstrapped resources: failed push remote to repository "git@github.com:*******/taxi-229.git" "Could not create directory '/.ssh'.\r\nHost key verification failed.\r\nfatal: Could not read from remote repository.\n\nPlease make sure you have the correct access rights\nand the repository exists.\n": exit status 128
```

From the error looks like it pointing to ```/.ssh``` location in the test container. I have merged this change to https://github.com/redhat-developer/kam/pull/229 to get some more clarity. I will update as per the behavior seen in test container.

Closing it.
/close